PendingReleaseNotes

```diff
@@ -205,3 +205,40 @@
   value determined by the config options, for the average for any of the 3 intervals.
   New admin command ``ceph daemon osd.# dump_osd_network [threshold]`` will
   do the same but only including heartbeats initiated by the specified OSD.
+
```

Comment:
instead of at the very bottom, this new release note should be added at the very top, under the heading

```
12.2.13
```

Reply:
ack. side effect of cherry-picking without proper thinking